.github/workflows/windows.yml

```diff
@@ -10,38 +10,44 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Cache Nasm install
-      uses: actions/cache@v1
-      id: cache-nasm
-      env:
-        cache-name: cache-nasm
-      with:
-        path: "C:\\Program Files\\Nasm"
-        key: cache-nasm-v1
-        restore-keys: cache-nasm-v1
     - name: Cache LLVM install
-      uses: actions/cache@v1
+      uses: actions/cache@v2
       id: cache-llvm
       env:
         cache-name: cache-llvm
       with:
         path: "C:\\Program Files\\LLVM"
         key: cache-llvm-v10.0.0.0-1
         restore-keys: cache-llvm-v10.0.0.0-1
+    - name: Cache vcpkg
+      uses: actions/cache@v2
+      id: cache-vcpkg
+      env:
+        cache-name: cache-vcpkg
+      with:
+        path: ${{ github.workspace }}/vcpkg
+        key: ${{ runner.os }}-build-${{ env.cache-name }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+    - name: Show vcpkg cache info
+      env:
+        VCPKG_INFO: ${{ toJSON(steps.cache-vcpkg) }}
+      run: echo "${VCPKG_INFO}"
+    - name: Has hit cache for vcpkg?
+      if: steps.cache-vcpkg.outputs.cache-hit == 'true'
+      run: echo Hit!
+    - name: Has missed cache for vcpkg?
+      if: steps.cache-vcpkg.outputs.cache-hit != 'true'
+      run: echo Missed!
```

Comment:
This seems as a debugging thing or am I wrong?

Reply:
I was thinking about letting it there at first, but to be honest...it is too situational to keep it there forever.

Reply:
Done.